core/core.go

```diff
@@ -65,7 +69,7 @@ type Driver interface {
 type NetworkDriver interface {
 	// A network driver implements the programming logic for network
 	Driver
-	Init(config *Config, stateDriver StateDriver) error
+	Init(config *Config, info *InstanceInfo, stateDriver StateDriver) error
```

Comment:
> should we call it Config and Runtime to keep it consistence with rest of the terminology

till now we have used `Runtime` for the oper state of something. Example, Config of a Driver, Runtime (or oper) of a Driver. But this info is more of the plugin-instance being passed to the Driver's Init(). So I think it is ok to call it InstanceInfo for now, until we have a need to pass non Instance specific things in future and then we may just call it `misc` :smile: .